Update grenade damage types to current usage

Turtlelord26 opened the pull request:
Primarily for #493. In accordance with #491, explosive damage references unaltered.
-all instances of burn damage changed to fire damage
-frag grenade primary damage tagged as ballistic
-instances of Poison replaced with Internal
-CID grenade damage typed as Internal
-Plasma grenade damage typed as plasma

-Wording update on C10 - the reflex paragraph referenced it as a grenade but it is not a grenade.

Turtlelord26 committed:
Update grenade damage types to current usage

Primarily for #493. In accordance with #491, explosive damage references unaltered.
-all instances of burn damage changed to fire damage
-frag grenade primary damage tagged as ballistic
-instances of Poison replaced with Internal
-CID grenade damage typed as Internal
-Plasma grenade damage typed as plasma

-Wording update on C10 - the reflex paragraph referenced it as a grenade but it is not a grenade.

bleehu commented on Items/04_01_Melee_&_Thrown_Weapons.txt:
```diff
@@ -362,14 +362,14 @@ at the cost of one action on their next turn.
-	Creates a 7m radius cloud of burning, choking smoke. Targets take 30 Poison 
-Damage per turn in the cloud. All Poison Damage is negated by use of a gas mask.
+	Creates a 7m radius cloud of burning, choking smoke. Targets take 30 Internal 
+Damage per turn in the cloud. All Internal Damage is negated by use of a gas mask.
```
Blocking; Poison damage wouldn't hurt cybernetic characters, Internal damage would.

Turtlelord26 replied:
Poison is not a valid damage type. Would it be satisfactory to write in that this grenade's (non-Impact) damage can only affect biological enemies?
This also implies a new feature request for an equivalent grenade that hurts robotic enemies in the same manner.

bleehu reviewed (changes requested):
One blocking issue; poison damage != internal damage in the case of cybernetic enemies.

rosebrallier reviewed (changes requested):
Question: Are we going to capitalize damage types or not? I notice that Plasma and Internal are while fire and ballistic are not. We should choose a single convention.

Turtlelord26 commented:
Since they only appear right before the word 'damage', I think we can get away with not capitalizing them from a clarity standpoint. I don't think they qualify as terms being used outside normal meanings either, so I'll go for uncapitalized.

Turtlelord26 committed:
Review updates

Turtlelord26 committed:
Merge branch 'dev' into Turtlelord26-#493-Grenade_Damage_Types

rosebrallier commented on Items/04_01_Melee_&_Thrown_Weapons.txt:
```diff
@@ -282,36 +282,36 @@ at the cost of one action on their next turn.
 
 	Everyone in the room gets blinded for 2 turns. Once per turn, You can 
-attempt to break the blindness with a perception check at 85% (disregarding 
+attempt to break the blindness with a DC 85 check (disregarding 
```
What kind of check? Cause making rolls that disregard bonuses plus don't have any bonuses to disregard in the first place isn't something I've seen much in CX.

bleehu replied:
Blocker: ya, we really need to know what kind of check this is.

Turtlelord26 replied:
That's a discussion issue then. The old stuff is nonfunctional enough that there are multiple ways to fix it and I'd rather not unilaterally pick one.
* DC X perception check (with Per mod)
* DC X spotting check (with spotting skill & per mod)
* DC X typeless check (i.e. no bonuses only flat roll, requires rules support in skills section of basic)
* Other

bleehu replied:
Spotting or awareness seems most likely to me.

rosebrallier replied:
Making it an Awareness save or a Perception check both work for me.

rosebrallier commented on Items/04_01_Melee_&_Thrown_Weapons.txt:
```diff
@@ -282,36 +282,36 @@ at the cost of one action on their next turn.
 	Everyone in the room gets blinded for 2 turns. Once per turn, You can 
-attempt to break the blindness with a perception check at 85% (disregarding 
+attempt to break the blindness with a DC 85 check (disregarding 
 bonuses). If the grenade detonates in the open, it’s blinding range is a 10m 
```
SC: "its"

rosebrallier commented on Items/04_01_Melee_&_Thrown_Weapons.txt:
```diff
@@ -282,36 +282,36 @@ at the cost of one action on their next turn.
 
-	Each character in the radius of a flashbang's range when it goes off may
+	Each character in the radius of a flash-bang's range when it goes off may
 make a Reflex Save of DC 110. If they succeed, they are able to shield their
```
Sc:Is Save capitalize?

rosebrallier commented on Items/04_01_Melee_&_Thrown_Weapons.txt:
```diff
@@ -282,36 +282,36 @@ at the cost of one action on their next turn.
-(if armor is pierced or not present): (3*45 damage)+2xBleed within a 2.5m radius.
-Only legs and torso (2*45 damage)+1xBleed within a 5m radius. At least one hit 
-must damage health to cause Bleed.
+	Each Target within the blast radius takes multiple hits. 45 ballistic 
```
Sc: clarity re taking multiple hits

rosebrallier commented on Items/04_01_Melee_&_Thrown_Weapons.txt:
```diff
@@ -282,36 +282,36 @@ at the cost of one action on their next turn.
+damage per hit, automatically hits armor. Targets take full damage to the torso, 
+legs, and arms (if armor is pierced or not present): (3*45 damage)+2xBleed 
+within a 2.5m radius.Only legs and torso (2*45 damage)+1xBleed within a 5m 
+radius. At least one hit must damage health to cause Bleed.
```
Sc: status effects capitalize?

rosebrallier reviewed (commented):
I'm scope creeping all over the place so I can find these moments later.

rosebrallier commented on Items/04_01_Melee_&_Thrown_Weapons.txt:
```diff
@@ -261,16 +261,16 @@ at the cost of one action on their next turn.
 the following effects: partial blindness due to uncontrollable tears (+30% to 
-hit), coughing fits (-50% stealth), and take 10 damage per round spent in the 
-cloud. Targets can negate all effects with a gas mask. Targets who leave the 
+hit), coughing fits (-50% stealth), and take 10 Internal damage per round spent in 
```
Internal

Turtlelord26 replied:
?

rosebrallier replied:
Ahem. "internal damage"

rosebrallier reviewed (approved):
Yay pending bleehu

Turtlelord26 committed:
Typo

bleehu reviewed (changes requested):
Small blocking change on flashbang save. Once added, you may shipit without further review.

Turtlelord26 committed:
Flashbang save update.

Now an Awareness save at the old DC, rate-limited to once per round as before but with better templating.
Save at the end of turn to avoid giving players two saves against the grenade before their first actions.

Turtlelord26 commented:
Still have to ask for a shipit, according to our rules for declared blocking issues.

rosebrallier reviewed (approved):
Shipit!

bleehu reviewed (approved):
Make shipit tha thing! Well done, Turtle!
